.github/ci-control/ci-config.json

```diff
@@ -0,0 +1,21 @@
+{
+    "__comment": [
+        "Compose and Main are two configurations we have in CI.",
+        "Each can have include and exclude filters where these filters run a",
+        "  'contains' match on the project name. e.g. if the project name is compose-runtime",
```

Comment:
Can we go with exact matches and validate it matches a real project? I am a bit worried this will lead to subtle errors.

Reply:
yea i flip-flopped here a lot on what to do. Another option is to allow matchers here (e.g. you can write `*compose*`).